src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -158,6 +167,13 @@ before_credit( void * _ctx,
   for( ulong i=0; i<ctx->xsk_aio_cnt; i++ ) {
     fd_xsk_aio_service( ctx->xsk_aio[i] );
   }
+
+  long now = fd_log_wallclock();
```

Comment:
This should be in housekeeping function, not `before_credit` (this is doing fd_log_wallclock every iteration of the runloop)

Reply:
how often does housekeeping get called?

Reply:
It's variable, but typically every few millis.